RiotSwiftUI/Modules/LocationSharing/LiveLocationSharingViewer/View/LiveLocationSharingViewer.swift

```diff
@@ -178,3 +190,27 @@ struct LiveLocationSharingViewer_Previews: PreviewProvider {
         }
     }
 }
+
+struct CenterToUserLocationButton: View {
```

Comment:
**Optional**
Shorter name are better imho.
```suggestion
struct LocalizationButton View {
```

Reply:
Not entirely agree since it can be misleading in future use